components/chart/legend.md

```diff
@@ -0,0 +1,93 @@
+---
+title: Legend
+page_title: Chart Legend
+description: Discover the possibilities of the Legend for the Blazor Chart. Learn how to show it, and explore the customization options.
+slug: chart-legend
+tags: telerik,blazor,chart,legend,customizations
+published: True
+position: 100
+---
+
+# Telerik Chart Legend
+
+The Telerik Chart for Blazor has the capability to showcase a visual guide offering details about the elements depicted in the chart. This article explores how to add a Chart legend, identify its building blocks, and customizing the legend.
+
+## Adding a Legend
+
+1. Add the `<ChartLegend>` child tag and set the `Visible` parameter to `true`
+1. Add the `Name` parameter to all Chart series that must be visible in the legend.
+
+## Chart Legend Customization
+
+You can customize the legend of the Telerik Chart for Blazor by adding nested (child) tags to the `<ChartLegend>` and use their parameters for fine-tuning. 
+
+The structure of the nested tags is `<ChartLegend*Specifics*>`, where the specifics can be:
+
+* `Title`
+* `Item`
+* `Border`
+
+>note Use the IntelliSense to explore the nested tags.
+
+## Chart Legend Series Customization
```

Comment:
This heading doesn't make it clear what is the relationship between the series and the legend. Consider something like

```suggestion
## Legend Settings in the Chart Series
```